Projects/19-Captsone_01/ispy1/inferential_statistics.py

```diff
@@ -0,0 +1,132 @@
+import pandas as _pd
```

Comment:
I'm guessing you're using `_pd` to get around the issue we previously discussed? It can be a bit confusing since `pd` is the convention that the majority of the community uses.